src/custom-elements/src/components/tabs/tabs.tsx

```diff
@@ -363,6 +363,25 @@ export class Tabs {
     }
   }
 
+  setAvailableSpaceAsync() {
+    return new Promise((resolve) => {
+      const startTime = Date.now();
+      const waitLimit = 1000; // milliseconds
```

Comment:
I think 'timeout' would be more descriptive.

Reply:
Thanks for pointing it out! @krzysztofdrozdowskilumen 

Actually, it's not a 'timeout'.
It's time limit, so the function is executed no more than the time limit.

To my mind, 'timeout' is more about an opposite - delay, break or suspension.

Probably, `limit`, `timeLimit`, `executionLimit` or `maxWorkTime` would be more descriptive
But :laughing: